pkg/monitor/monitor/util.go

```diff
@@ -389,17 +373,13 @@ func getMonitorInitContainer(monitor *v1alpha1.TidbMonitor, tc *v1alpha1.TidbClu
 	return container
 }
 
-func getMonitorDMInitContainer(monitor *v1alpha1.TidbMonitor, dc *v1alpha1.DMCluster) core.Container {
+func getMonitorDMInitContainer(monitor *v1alpha1.TidbMonitor) core.Container {
 	// TODO: Support dm in reloader. Currently dm cluster shares the same persistent rules dir with tidb cluster
 	command := getInitCommand(monitor)
 	container := core.Container{
 		Name:  "dm-initializer",
 		Image: fmt.Sprintf("%s:%s", monitor.Spec.DM.Initializer.BaseImage, monitor.Spec.DM.Initializer.Version),
 		Env: []core.EnvVar{
-			{
```

Comment:
The DM envs are also used in https://github.com/pingcap/ticdc/blob/master/dm/monitoring/init.sh, we have to update that script first.

Reply:
BTW, the `DM_VERSION` is retrieved from the TiDB initializer version instead of DM initializer.